src/payment/strategies/ppsdk/ppsdk-strategy.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { createRequestSender } from '@bigcommerce/request-sender';
```

Comment:
Given that this strategy currently dead-ends to dummy handlers (`stepHandler` and `PaymentProcessor`) and the overall feature is still a WIP, this test seemed like enough to cover the entire strategy file for now.